packages/lab/src/menu-button/internal/MenuTrigger.css

```diff
@@ -1,6 +1,6 @@
 .uitkMenuTrigger {
-  --button-open-color: var(--uitkMenuTrigger-button-open-color, var(--uitk-actionable-primary-foreground-active));
-  --icon-marginLeft: var(--uitkMenuTrigger-icon-marginLeft, var(--uitk-size-unit));
+  --button-open-color: var(--uitkMenuButton-trigger-button-open-color, var(--uitk-actionable-primary-foreground-active));
+  --icon-marginLeft: var(--uitkMenuButton-trigger-icon-marginLeft, var(--uitk-size-unit));
```

Comment:
should the MenuTrigger component be called Trigger then?

Reply:
good point - and moved to no longer internal since it has CSS exposed